modules/claims_api/app/sidekiq/claims_api/evidence_waiver_builder_job.rb

```diff
@@ -40,15 +46,32 @@ def benefits_doc_api
     def benefits_doc_upload(claim:, pdf_path:, doc_type:, ptcpnt_vet_id:)
       if Flipper.enabled?(:claims_api_ews_uploads_bd_refactor)
         EvidenceWaiverDocumentService.new.create_upload(claim:, pdf_path:, doc_type:, ptcpnt_vet_id:)
+        log_ews_event(claim, :benefits_doc_upload,
+                      'EvidenceWaiverDocumentService creation successful.')
       else
         benefits_doc_api.upload(claim:, pdf_path:, doc_type:, pctpnt_vet_id: ptcpnt_vet_id)
+        log_ews_event(claim, :benefits_doc_upload, 'BD API upload successful.')
       end
     end
 
     def update_status_for_submission(lighthouse_claim)
       pending_state = ClaimsApi::EvidenceWaiverSubmission::PENDING
 
-      set_state_for_submission(lighthouse_claim, pending_state) if lighthouse_claim.status != pending_state
+      if lighthouse_claim.status != pending_state
+        set_state_for_submission(lighthouse_claim, pending_state)
+        log_ews_event(lighthouse_claim, :update_status_for_submission,
+                      "Status updated from #{lighthouse_claim.status} to #{pending_state}.")
+      end
+    end
+
+    private
+
+    def log_ews_event(claim, method_name, detail, **)
+      ClaimsApi::Logger.log('EWS_builder',
+                            ews_id: claim&.id,
+                            claim_id: claim&.claim_id,
+                            detail: "EvidenceWaiverBuilderJob##{method_name}: #{detail}",
```

Comment:
What do you think about making `method_name` its own key instead of combining it with `detail`? There are only a few keys that change the logging behavior (e.g. `level`, which sets the rails logger level). Since we're already tagging the log with `EWS_builder` I think we can skip outputting the class as well.